package.json

```diff
@@ -4,5 +4,9 @@
   "private": true,
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "scripts": {
+    "build": "yarn workspaces foreach --parallel --interlaced --verbose --topological run build",
```

Comment:
So this relates to a somewhat related thread I started with @bencodrington, do we actually want to build (e.g. transcompile) the serve code? Or are we happy running the server through ts-node/ts-node-esm executables? 

Any thoughts?

Reply:
For now I just added main scripts to the project, I think their names should match into which environment we want to run.

Reply:
If that's the case we'll need to do some further configuration. Currently `tsc` fails in the `server` package because it's not able to follow the link to the `shared` package.

Reply:
So currently the trans-compilation is not working. The `shared` module is not packaged correctly server-side.